site/assets/scss/_navbar.scss

```diff
@@ -1,8 +1,16 @@
 .bd-navbar {
   padding: .75rem 0;
   background-color: transparent;
-  background-image: linear-gradient(to bottom, rgba(var(--bd-violet-rgb), 1), rgba(var(--bd-violet-rgb), .95));
-  box-shadow: 0 .5rem 1rem rgba(0, 0, 0, .15), inset 0 -1px 0 rgba(0, 0, 0, .15);
+  box-shadow: 0 .5rem 1rem rgba($black, .15), inset 0 -1px 0 rgba($white, .15);
+
+  &::after {
+    position: absolute;
+    inset: 0;
+    z-index: -1;
+    display: block;
+    content: "";
+    background-image: linear-gradient(to bottom, rgba(var(--bd-violet-rgb), 1), rgba(var(--bd-violet-rgb), .95));
```

Comment:
Just for information, this `linear-gradient` (same thing for `main`) doesn't work before Big Sur (Safari 14).
Here is the rendering with Catalina (Safari 13.1):

![Screenshot 2022-11-24 at 13 20 59](https://user-images.githubusercontent.com/17381666/203783456-72c799bf-72a4-49fd-9c54-eb428afca87b.png)